libs/scripts/scripts.js

```diff
@@ -149,7 +157,31 @@ const eagerLoad = (img) => {
 
 (async function loadPage() {
   performance.mark('loadpage');
-  setConfig(config);
+  const newConfig = setConfig(config);
   loadLana({ clientId: 'milo' });
+
+  setupRUM();
+
+  await initAnalyticsTrackingQueue();
+
+  const cwv = {};
+
+  // Forward the RUM CWV cached measurements to edge using WebSDK before the page unloads
+  window.addEventListener('beforeunload', () => {
+    if (!Object.keys(cwv).length) return;
+    analyticsTrackCWV(cwv);
+  });
+
+  // Callback to RUM CWV checkpoint in order to cache the measurements
+  sampleRUM.always.on('cwv', async (data) => {
```

Comment:
@iuliag i take it this is informational only and no action is required?

Reply:
Yes, informational, so that who builds the reports in AA to put a note there and not expect every page view to have CWVs recorded.
As part of the RUM - AA/CJA discussions for conversion tracking, we're looking into how to properly delineate between sampled and non-sampled data.